ts/components/messages/MessageCTABar.tsx

```diff
@@ -133,6 +134,10 @@ class MessageCTABar extends React.PureComponent<Props, State> {
     const { message, calendarEvent, showModal, disabled } = this.props;
     const { isEventInCalendar } = this.state;
 
+    const calendarIconComponentSize = this.props.small
+      ? { width: "32", height: "32" }
```

Comment:
why we need two values if they're the same?

```suggestion
      ? 32 : 48
```

Reply:
changed